tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network.md

```diff
@@ -0,0 +1,228 @@
+    ```
+
+2. In the `[features]` section, add the Utility pallet to the `std` feature list by including:
+    ```toml
+    [features]
+    default = ["std"]
+    std = [
+        "codec/std",
+        "scale-info/std",
+        "frame-executive/std",
+        ...
+        "pallet-utility/std",
+    ]
+    ```
+
+3. Save the changes and close the `Cargo.toml` file
+
+### Update the Runtime Configuration
+
+To implement the necessary changes in your runtime, you'll need to modify the `runtime/src/lib.rs` file.
+
+#### Configure the Utility Pallet
+   
+1. Implement the [`Config`](https://paritytech.github.io/polkadot-sdk/master/pallet_utility/pallet/trait.Config.html){target=\_blank} trait for the Utility pallet:
+
+    ```rust
+    --8<-- 'code/tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network/config-trait-impl.rs'
+    ```
+
+2. Locate the `#[frame_support::runtime]` macro and add the Utility pallet:
+
+    ```rust
+     --8<-- 'code/tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network/fs-runtime-macro.rs'
+    ```
+
+#### Update Existential Deposit Value
+
+To update the `EXISTENTIAL_DEPOSIT` in the Balances pallet, locate the constant and set the value to `1000`:
+
+```rust
+...
+/// Existential deposit.
+pub const EXISTENTIAL_DEPOSIT: u128 = 1000;
+...
+```
+
+!!!note
+    This change increases the minimum balance required for accounts to remain active. No accounts with balances between `500` and `1000` will be removed. For account removal, a storage migration is needed. See [Storage Migration]() for details.
+
+#### Update Runtime Version
+
+Locate the `runtime_version` macro and increment the `spec_version` field from `100` to `101`:
+
+```rust
+--8<-- 'code/tutorials/polkadot-sdk/build-a-blockchain/upgrade-a-running-network/runtime-version-macro.rs'
+```
+
+### Recompile the Runtime
+
+Once you've made all the necessary changes, recompile the runtime by running:
+
+```bash
+cargo build --release
+```
+
+The build artifacts will be output to the `target/release` directory. The Wasm build artifacts can be found in the `target/release/wbuild/solochain-template-runtime` directory. You should see the following files:
+
+- *solochain_template_runtime.compact.compressed.wasm*
+- *solochain_template_runtime.compact.wasm*
+- *solochain_template_runtime.wasm*
+
+## Execute the Runtime Upgrade
+
+Now that you've generated the Wasm artifact for your modified runtime, it's time to upgrade the running network. This process involves submitting a transaction to load the new runtime logic.
+
+### Understand Runtime Upgrades
+
+#### Authorization with Sudo
+
+In production networks, runtime upgrades typically require community approval through governance. For this tutorial, the Sudo pallet will be used to simplify the process. The Sudo pallet allows a designated account (usually `Alice` in development environments) to perform privileged operations, including runtime upgrades.
```

Comment:
```suggestion
In production networks, runtime upgrades typically require community approval through governance. For this tutorial, the Sudo pallet will be used to simplify the process. The Sudo pallet allows a designated account (usually Alice in development environments) to perform privileged operations, including runtime upgrades.
```

I don't think Alice needs to be in backticks because it's referring to the account not code